### PR TITLE
Fix Avatar Issue #240

### LINKED ISSUE #240: Replace fuselage's Avatar with our Avatar component in RoomInformation
We have already built our own Avatar Component. It needs to be replaced with the fuselage's Avatar component used in the RoomInformation component.
This could be a good first issue for new contributors.

### PR DESCRIPTION
**Pull request by KushalSr**
# Replace fuselage's Avatar with our Avatar component in RoomInformation

- Commented-  //import { Avatar } from '@rocket.chat/fuselage'; 
- Added  - import { Avatar } from '../Avatar/Avatar'

Fix #240

### CODE REVIEW

File: packages/react/src/components/RoomInformation/RoomInformation.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useContext } from 'react';
-import { Avatar } from '@rocket.chat/fuselage';
+//import { Avatar } from '@rocket.chat/fuselage';
```
Comment (abhinavkrin) [inline packages/react/src/components/RoomInformation/RoomInformation.js:2]: Hey @KushalSr You can safely remove this comment.

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
The comment could be safely removed.

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
The comment could be safely removed.

File: packages/react/src/components/RoomInformation/RoomInformation.js (hunk 1 above)
Comment (KushalSr) [reply packages/react/src/components/RoomInformation/RoomInformation.js:2]: Hi @abhinavkrin , actually I'm having some issues with my laptop. Can you wait till evening, I'll remove the comment or if it's urgent then I've to use someone's else laptop.
Thanks

Comment (abhinavkrin) [reply packages/react/src/components/RoomInformation/RoomInformation.js:2]: Hey @KushalSr no worries take your time.

Comment (KushalSr) [reply packages/react/src/components/RoomInformation/RoomInformation.js:2]: Thanks @abhinavkrin

Comment (KushalSr) [reply packages/react/src/components/RoomInformation/RoomInformation.js]: @abhinavkrin Hi Abhinav. I've remove the comment check once.
Thanks

**Review by abhinavkrin — APPROVED** [review]
LGTM! Thank you!

File: packages/react/src/components/RoomInformation/RoomInformation.js (hunk 1 above)
Comment (abhinavkrin) [reply packages/react/src/components/RoomInformation/RoomInformation.js]: Hey @KushalSr ,  Have you checked if the image of the room is being rendered or not?

Comment (abhinavkrin) [reply packages/react/src/components/RoomInformation/RoomInformation.js]: The image of the room is not being rendered. Could you set size="100%" in line 55 , like `<Avatar size="100%" url={getChannelAvatarURL(channelInfo.name)} />`

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
Please sign the CLA (Contributor License agreement).
The avatar is not rendering. The size should be put to 100%.

File: packages/react/src/components/RoomInformation/RoomInformation.js (hunk 1 above)
Comment (KushalSr) [reply packages/react/src/components/RoomInformation/RoomInformation.js]: @abhinavkrin okay, I'll work on that

**Review by abhinavkrin — CHANGES_REQUESTED** [review]
Still, Avatar from fuselage is being used.

**Review by abhinavkrin — APPROVED** [review]
LGTM! Thank you

**Pull request merged**